scripts/buildWebsite.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const tempy = require('tempy');
+const fs = require('fs-extra');
+const packageJSON = require('../package.json');
+const path = require('path');
+const glob = require('fast-glob');
+const {spawn, execSync} = require('child_process');
+
+build().catch(err => {
+  console.error(err.stack);
+  process.exit(1);
+});
+
+async function build() {
+  // Create a temp directory to build the site in
+  let dir = tempy.directory();
+  console.log(`Building into ${dir}...`);
+
+  // Generate a package.json containing just what we need to build the website
+  let pkg = {
+    name: 'rsp-website',
+    version: '0.0.0',
+    private: true,
+    workspaces: [
+      'packages/*/*'
+    ],
+    devDependencies: Object.fromEntries(
+      Object.entries(packageJSON.devDependencies)
+        .filter(([name]) =>
+          name.startsWith('@parcel') ||
+          name === 'parcel' ||
+          name === 'patch-package' ||
+          name.startsWith('@spectrum-css') ||
+          name.startsWith('postcss')
+        )
+    ),
+    dependencies: {},
+    resolutions: packageJSON.resolutions,
+    browserslist: packageJSON.browserslist,
+    scripts: {
+      build: "PARCEL_WORKER_BACKEND=process parcel build 'docs/*/*/docs/*.mdx' 'packages/dev/docs/pages/**/*.mdx' --no-scope-hoist",
```

Comment:
cc @mischnic had to turn off scope hoisting here. The ListBox page throws an error: `Uncaught ReferenceError: $a5f3adf7483dfc2b62c1ecfb0311eae$export$Section is not defined`. I didn't have a chance to look into it yet.